subprojects/packagefiles/croaring/meson.build

```diff
@@ -36,9 +39,13 @@ croaring_lib = library(
   'croaring',
   croaring_sources,
   include_directories: croaring_headers,
+  override_options: ['c_std=c11', 'cpp_std=c++11'],
```

Comment:
hrm I checked this project out.

So the compiled files are C only. The headers are what seem to have C++ support. Which means c++11 is not needed here.